plugins/inputs/procstat/procstat_linux.go

```diff
@@ -0,0 +1,111 @@
+//go:build linux
+// +build linux
+
+package procstat
```

Comment:
Which part is "linux" specific here? I would rather like to avoid splitting `procstat.go` into OS-specific stuff.